### PR TITLE
Update Getting Started

### PR DESCRIPTION
**Pull request by syuhei176**
* change wallet repo
* add enter private key

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,13 +112,13 @@ You need envs described [here](https://github.com/cryptoeconomicslab/plasma-cham
...
 ```sh
-git clone https://github.com/cryptoeconomicslab/plasma-wallet
+git clone https://github.com/cryptoeconomicslab/wallet
 yarn install
 cp .env.example .env
 yarn start
 ```
 
-Open http://localhost:1234 in browser.
+Open http://localhost:1234 in browser and enter your private key.
```
Comment (tkmct) [inline README.md]: localhost:8080 is the default port for this app. sorry for that.  
: (

Comment (syuhei176) [reply README.md]: ah OK! can you write getting started for your wallet?

Comment (tkmct) [reply README.md]: Sure!!
Then we can delegate this part to it.

**Comment by shogochiai** [comment]
@tkmct What is remaining task for this one? Please warmly share us that, or one of us inherit your task :)

**Pull request closed without merging**